### PR TITLE
added button text field to change the optibp button text

### PR DESCRIPTION
**Pull request by junaidwarsivd**
related issue https://github.com/opensrp/opensrp-client-native-form/issues/614

### CODE REVIEW

**Comment by SebaMutuku** [comment]
@ndegwamartin  would you mind having a look at this?

**Review by SebaMutuku — COMMENTED** [review]
@bennsimon  would you mind having a look at this?

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/OptiBPWidgetFactory.java [hunk 1]
```diff
@@ -164,6 +164,12 @@ private void formatButtonWidget(Button button, JSONObject jsonObject) throws JSO
             String colorString = jsonObject.getString(JsonFormConstants.OptibpConstants.OPTIBP_KEY_BUTTON_TEXT_COLOR);
             button.setTextColor(Color.parseColor(colorString));
         }
+        if(jsonObject.has(JsonFormConstants.OptibpConstants.OPTIBP_KEY_BUTTON_TEXT))
```
Comment (SebaMutuku) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/OptiBPWidgetFactory.java:167]: Format this section 
 if(jsonObject.has(JsonFormConstants.OptibpConstants.OPTIBP_KEY_BUTTON_TEXT))
        {
            String buttonText = jsonObject.getString(JsonFormConstants.OptibpConstants.OPTIBP_KEY_BUTTON_TEXT);
            button.setText(buttonText);
        }

Comment (junaidwarsivd) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/OptiBPWidgetFactory.java]: done @SebaMutuku

**Review by SebaMutuku — APPROVED** [review]

**Pull request merged**